### PR TITLE
Adding loader for CompMusic Raga Dataset

### PR DESCRIPTION
**Pull request by genisplaja**
### Description

Please include the following information at the top-level docstring for the dataset's module mydataset.py:

- [x] Describe annotations included in the dataset
- [x] Indicate the size of the datasets (e.g. number files and duration, hours)
- [x] Mention the origin of the dataset (e.g. creator, institution)
- [x] Describe the type of music included in the dataset
- [x] Indicate any relevant papers related to the dataset
- [x] Include a description about how the data can be accessed and the license it uses (if applicable)

#### Dataset loaders checklist:

- [x] Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file.
- [x] Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`.
- [x] Create a module in mirdata, e.g. `mirdata/my_dataset.py`
- [x] Create tests for your loader in `tests/datasets/`, e.g. `test_my_dataset.py`
- [x] Add your module to `docs/source/mirdata.rst` and `docs/source/table.rst`
- [x] Run `tests/test_full_dataset.py` on your dataset **(ongoing, it takes a while)**

At the moment there is no Zenodo entry for this dataset. We will create it soon, separating the audio (which must be restricted) and the annotations/features (which are open).

### CODE REVIEW

File: scripts/make_compmusic_raga_index.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+def make_otmm_makam_index(dataset_data_path):
+
+    raga_index = {"version": "1.0", "tracks": {}, "metadata": []}
+    dataset_folder = "RagaDataset"
+
+    traditions = ["Carnatic", "Hindustani"]
+    for trad in traditions:
+        tradition_name = trad
+        for raga in glob.glob(os.path.join(dataset_data_path, trad, "audio", "*/")):
+            raga_id = raga.split("/")[-2]
```
Comment (nkundiushuti) [inline scripts/make_compmusic_raga_index.py:21]: os.path.sep instead of "/"? otherwise this may fail on windows

**Review by nkundiushuti — CHANGES_REQUESTED** [review]

File: mirdata/datasets/compmusic_raga_dataset.py [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+REMOTES = {
+    "all": download_utils.RemoteFileMetadata(
+        filename="TODO",
+        url="TODO",
+        checksum="TODO",
+    )
+}
+
+DOWNLOAD_INFO = """While annotations and metadata are freely downloadable, the audio of this 
+    dataset has restricted access. Please access: LINK HERE and request access to the audio,
```
[the remark this answers is not in the document]
Comment (genisplaja) [reply mirdata/datasets/compmusic_raga_dataset.py:82]: before... will do the upload soon!

File: docs/source/table.rst [hunk 3]
```diff
@@ -80,6 +80,20 @@
...
 
+   * - .. line-block::
+
+        (CompMusic)
+        Raga Dataset
+
+     - - audio: 🔑
+       - annotations: ✅
+     - - :ref:`f0`
+       - :ref:`phrases`
```
[the remark this answers is not in the document]
Comment (genisplaja) [reply docs/source/table.rst:91]: Yes... maybe segments would be more appropriate, I'd need to create a new entry in quick reference, and `segments` is quite a broad concept, I hope that does not mean a problem.

File: scripts/make_compmusic_raga_index.py (hunk 1 above)
Comment (genisplaja) [reply scripts/make_compmusic_raga_index.py:21]: that is true! I'll change that.

Comment (genisplaja) [reply scripts/make_compmusic_raga_index.py:21]: We will actually address that given the refactor we need to do to fix path issues with windows all over mirdata.

**Review by nkundiushuti — APPROVED** [review]

**Comment by magdalenafuentes** [comment]
@genisplaja small request: can you change the name of the module from `compumusic_raga_dataset` to simply `compmusic_raga`? it will make our lives easier in the tables and for the user if the name is shorter

**Comment by genisplaja** [comment]
@magdalenafuentes done:) Was definitely not needed

**Pull request merged**